src/providers/ldap/sdap_async_groups.c

```diff
@@ -880,6 +880,8 @@ static int sdap_save_grpmem(TALLOC_CTX *memctx,
     int ret;
     const char *remove_attrs[] = {SYSDB_MEMBER, SYSDB_ORIG_MEMBER, SYSDB_GHOST,
                                   NULL};
+    const char *check_dom;
+    const char *check_name;
```

Comment:
@sumit-bose, sorry for nitpicking, but I'd prefer to have those two variables initialized as NULL.